src/tests.rs

```diff
@@ -223,9 +220,7 @@ fn test_rust_function() {
     ).unwrap();
 
     let lua_function = globals.get::<_, Function>("lua_function").unwrap();
-    let rust_function = lua.create_function(|lua, _| {
-        lua.pack("hello")
-    });
+    let rust_function = lua.create_function(|_, _: ()| Ok("hello"));
```

Comment:
I actually HAVE used it in let bindings and function arguments lots, I just had never used it as a closure parameter without a type, it was the lack of a type that was getting me somehow.  It makes sense now that I think about it.